care/users/management/commands/load_skill_data.py

```diff
@@ -0,0 +1,59 @@
+from django.core.management import BaseCommand
+
+from care.users.models import Skill
+
+
+class Command(BaseCommand):
+    """
+    Command to load default skills
+    """
+
+    help = "Seed Data for Skills"
+
+    def handle(self, *args, **options):
+        self.stdout.write("Seeding Skills Data... ", ending="")
+
+        skills = [
+            "Anesthetists",
+            "Cardiac Surgeon",
+            "Cardiologist",
+            "Dermatologist",
+            "Diabetologist",
+            "Emergency Medicine Physcian",
+            "Endocrinologist",
+            "Family Physcian",
+            "Gastroenterologist",
+            "General Medicine",
+            "General Surgeon",
+            "Hematologist",
+            "Intensivists",
+            "Medical Officer",
+            "Nephrologist",
+            "Neuro Surgeon",
+            "Neurologist",
+            "obstetrician and Gynecologists",
+            "Oncologist",
+            "Oncology Surgeon",
+            "Opthalmologists",
```

Comment:
I see a lot of plural words, can we switch to singular form?

Reply:
cc: @gigincg @aparnacoronasafe